shared/test/test_shared_all.py

```diff
@@ -10,7 +10,7 @@ class TestSharedUnits(BaseTestFrontendUnits):
 
     def setUp(self):
         super(TestSharedUnits, self).setUp()
-        self.set_server_prefix(".")
+        self.set_server_prefix("../../../../")
```

Comment:
A comment about what this default assumes or is going to resolved against would be nice; similar suggestion for the other use.